index.d.ts

```diff
@@ -17,33 +17,61 @@ export interface SnapshotOptions {
 	id?: string;
 }
 
+// When an assert fail, the message string will be shown in the error
 export interface Assertions {
+	// Assert that actual is deep equal to expected
 	deepEqual<ValueType = any>(actual: ValueType, expected: ValueType, message?: string): void;
+	// Failing assertion
 	fail(message?: string): void;
+	// Assert that actual is equal to false
 	false(actual: any, message?: string): void;
+	// Assert that actual is falsy (false, 0, '', "", null, undefined, NaN)
 	falsy(actual: any, message?: string): void;
+	// Assert that error is falsy
 	ifError(error: any, message?: string): void;
+	// Assert that actual is equal to expected (use deepEqual for objects/arrays)
 	is<ValueType = any>(actual: ValueType, expected: ValueType, message?: string): void;
+	// Assert that actual is not equal to expected (use notDeepEqual for objects/arrays)
 	not<ValueType = any>(actual: ValueType, expected: ValueType, message?: string): void;
+	// Assert that actual is not deep equal to expected
 	notDeepEqual<ValueType = any>(actual: ValueType, expected: ValueType, message?: string): void;
+	// Assert that string does not follow the regex pattern
 	notRegex(string: string, regex: RegExp, message?: string): void;
+	// Assert that a function that never returns does not throw an error
```

Comment:
The "never returning" part is a type detail that's not relevant to the documentation. It's so that you can write `t.notThrows(() => { throw new Error() })` without type errors.